src/main/java/com/esri/arcgisruntime/toolkit/skins/feature-template-picker.css

```diff
@@ -0,0 +1,45 @@
+.feature-template-picker {
+  -fx-spacing: 0;
+}
+
+.feature-template-picker:horizontal .feature-template-group {
+  -fx-border-width: 0 1 0 0;
+}
+
+.feature-template-group {
+  -fx-spacing: 5;
+  -fx-padding: 5;
+  -fx-alignment: top-center;
+  -fx-border-color: -fx-box-border;
+  -fx-border-width: 0 0 1 0;
+}
+
+.feature-template-item {
+  -fx-text-fill: black;
+  -fx-base: transparent;
+  -fx-background: transparent;
+  -fx-focus-color: transparent;
+  -fx-border-color: transparent;
+  -fx-effect: null;
+  -fx-content-display: top;
+  -fx-min-width: 80;
+}
+
+.feature-template-item:hover {
+  -fx-background-color: rgba(10, 10, 10, 0.1);
```

Comment:
Ended up just removing these to use the toggle buttons default colors. The only one that might need some change is the hover state because it's pretty close to the `-fx-base` background color